docs/config.md

```diff
@@ -1,7 +1,6 @@
 # Configuration
 
-You can configure the app by copying `config.sample.json` to `config.json` and customising it. The possible options are
-described here. If you run into issues, please visit [#element-web:matrix.org](https://matrix.to/#/#element-web:matrix.org)
+You can configure the app by making a copy of the `config.sample.json` and renaming it as `config.json` within the same directory. You can also customize it. The possible options are described here. If you run into issues, please visit [#element-web:matrix.org](https://matrix.to/#/#element-web:matrix.org)
```

Comment:
For this documentation, `config.sample.json` is not necessarily at the same level or place as `config.json`. The existing language to say copying the sample to the `config.json` file felt clear enough? 

"You can also customize it." is also somewhat irrelevant when looking at this doc - I think that sentence can be removed.

Reply:
I meant the `config.json` file to be in the same folder as the `config.sample.json` (element-web folder).

The existing language implied there was already a config,json file. I had issues interpreting it as well as a few other people from the  [#element-web:matrix.org](https://matrix.to/#/#element-web:matrix.org) group chat. I thought it would be better to clarify it for new contributors.